recipes/cifkit/meta.yaml

```diff
@@ -32,21 +32,21 @@ requirements:
 
 test:
   imports:
-    - cifkit
+    - {{ name }}
   commands:
     - pip check
   requires:
     - pip
 
 about:
-  home: https://github.com/bobleesj/cifkit
+  home: https://github.com/bobleesj/{{ name }}
```

Comment:
Please don't use jinja variable here. It doesn't help maintain the recipe but it does obscure it and render the URL unclickable.

Reply:
@ocefpaf  Thank you! No variables are used for any of the URLs.

Please review!